ark/phenotyping/som_runner.R

```diff
@@ -0,0 +1,36 @@
+library(data.table)
+library(FlowSOM)
+
+print("Reading command args")
+args <- commandArgs(trailingOnly=TRUE)
+
+# get path to pixel mat data
+print("Getting pixel mat data path")
+pixelMatPath <- args[1]
+
+# create a vector out of the list of channels provided
+print("Creating marker list")
```

Comment:
To me it makes more sense to do this step before saving the CSV file. 

But, if people want to try out different markers, it might be better to save everything in the CSV and then just subset to this function. Not sure.

Reply:
I'm specifying the marker list now to make it easy to read the CSV data in with only the markers we need before training the SOM. Would it be better to save to a CSV before training the SOM? I'm not entirely sure what the aim is here.